tests/integration/product-type-export-encoding.spec.js

```diff
@@ -112,45 +79,46 @@ test(`productType export module
   const expectedEncoded2 = 'name,key,description,breite\ncustom-product-type,'
     + 'productTypeKey,Some description - �lu�ou�k� k�� �p�l ��belsk� �dy,X\n'
 
-  before([productTypeKey]).then(() => productTypeExport.run())
-  .then(() =>
-    new Promise((resolve) => {
-      glob(path.join(OUTPUT_FOLDER, '*'), (err, files) => {
-        t.equal(files.length, 2, 'files length is 2')
-        t.equal(
-          files[0].split('/').pop(),
-          expectedFileName1,
-          `file name is ${expectedFileName1}`,
-        )
-        t.equal(
-          files[1].split('/').pop(),
-          expectedFileName2,
-          `file name is ${expectedFileName2}`,
-        )
-        resolve()
+  return before()
```

Comment:
+1 For going the async way above. We can also await on before, to make the code even more compact

Reply:
Most of tests updated here 0a632e9